README.md

```diff
@@ -54,9 +54,9 @@ For more details on the yotta/mbed OS edition of mbed TLS, including example pro
 
 ### Make
 
-We intentionally only use the minimum of `Make` functionality, as a lot of `Make` features are not supported on all different implementations of Make or on different platforms. As such, the Makefiles sometimes require some manual changes or export statements in order to work for your platform.
+We require GNU Make. To build the library and the sample programs, GNU Make and a C compiler are sufficient. Some of the more advanced build targets require some Unix/Linux tools.
```

Comment:
Hi @gilles-peskine-arm - I think we're reading this paragraph differently and where I'm seeing something general, I think you're seeing something more precise, although I must admit, I still think we disagree.

I think the spirit of the original `Makefile` that Paul originally set out was to make it simple and easy to adapt to your own platform and your own tools. Lots of people take mbed TLS and adapt it in their own way, and I do not want to stray far from this approach.

Yes, "in principle anything can be made to work if you make enough changes", but I'd like to minimise the number of those changes.

I don't mind declaring we test only with GNU, that the `Makefile` uses GNU specific features, but I don't see the need to give up on that original approach of keeping it simple. If we only wanted to support one build system, we'd likely stick with CMake, and give up on maintaining this second Makefile.

So having said that, how would you rephrase the paragraph above, that I suggested adding back in.